admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/DynamicFieldServiceImpl.java

```diff
@@ -116,17 +111,21 @@ public class DynamicFieldServiceImpl implements DynamicFieldService {
 			condition="#langCode != null")
 	@Override
 	public PageDto<DynamicFieldExtnDto> getAllDynamicField(int pageNumber, int pageSize, String sortBy, String orderBy, String langCode,
-															   LocalDateTime lastUpdated, LocalDateTime currentTimestamp) {
+															   LocalDateTime lastUpdated, LocalDateTime currentTimestamp, String fieldNameOptional) {
```

Comment:
@kameshsr i think it will be better & clean way to create new method to get data specific to fieldName. Lets create a new method.

Reply:
Yes done.